### DISCUSSION TITLE
How to use readonly in RichEditor

### DISCUSSION THREAD

**Discussion by khoait03** [filamentphp/filament#14864] [Help]
### Package

Form builder

### Package Version

v3.4.18

### How can we help you?

**I used this method but it didn't work**

RichEditor::make('content')
      ->label('Nội dung email')
      ->required()
      ->reactive()
      ->placeholder('Nội dung sẽ tự động cập nhật...')
      ->extraAttributes(['readonly' => true])
      ->default(function () use ($record) {

          return '<h3>Chọn trạng thái ứng tuyển để hiển thị nội dung email</h3>';
      }),

**Comment by leandrocfe** [comment]
try
```php
RichEditor::make('content')
...
->extraInputAttributes(['contentEditable' => 'false'])
```

**Reply by khoait03** [reply]
Thanks, it worked
